mayastor/src/grpc/mod.rs

```diff
@@ -33,6 +33,40 @@ mod mayastor_grpc;
 mod nexus_grpc;
 mod server;
 
+use crate::{bdev::VerboseError, subsys};
+use futures::Future;
 pub use server::MayastorGrpcServer;
 use tonic::{Response, Status};
+
 pub type GrpcResult<T> = std::result::Result<Response<T>, Status>;
+
+/// exports the current configuration to the mayastor config file
+fn export_config() -> std::result::Result<(), Status> {
```

Comment:
this function could also be moved to subsys/config -- just to keep it all organized and clean. Look at how to do that in  mod file, look for  pub use XXX -- thats where the visiblity is controled mostly.

Reply:
Moved